public/js/index_datatables.js

```diff
@@ -58,7 +58,7 @@ function initIndex(pagesize) {
 	});
 
 	//add datatable search event to the local searchbox and clear search to the clear filter button
-	$('#srch').keyup(function () {
+	$('#subsrch').click(function () {
```

Comment:
I'd like for this to trigger on the Enter button as well: Converting the entire thing to `<form>` is a solution, but you can just add a keyup on the search folder that only triggers `performSearch` if the key is Enter.